core/_cli_utils.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import click
+from . import utils
+
+
+logger = logging.getLogger(__name__)
+
+class opts:
+    layout = click.option(
+        "--layout", default=None, type=click.Choice(list(utils.ANTS_DICT.keys())),
+        help="A pre-defined HERA layout to use"
+    )
+    ants = click.option(
+        "-a", "--ants", type=click.IntRange(0, 350), default=None, multiple=True,
+        help="ants to use as a subset of the full HERA 350"
+    )
+    freq_range = click.option(
+        "-fr",
+        "--freq-range",
+        nargs=2,
+        default=(0, len(utils.FREQS_DICT['H4C'])),
+        show_default=True,
+        metavar="START STOP",
+        type=click.IntRange(0, len(utils.FREQS_DICT['H4C'])),
+        help="Frequency channel range (zero-based)",
+    )
+    freqs = click.option(
+        "-fch",
+        "--freqs",
```

Comment:
`freq-range` and `freqs` usually make me think of floating point frequencies. `fch` make senses. Maybe we should just emphasis somewhere that the frequency selection is done by channel numbers exclusively.

Reply:
OK, I fixed this by using `--channels` and also allowing `--freq-range` to be actually a frequency range in MHz